JitsiMeetJS.ts

```diff
@@ -538,7 +538,6 @@ export default {
                     domain: jaasDomain,
                     muc: `conference.${appId}.${jaasDomain}`
                 },
-                conferenceRequestUrl: `https://${jaasDomain}/${appId}/conference-request/v1?room=${roomName}`,
```

Comment:
Oh I see that the LJM getting started sample was modified to use JitsiMeetJS.joinConference() instead of JitsiConference.join(). Unfortunately, we cannot delete this line here. Maybe we need to keep both modes in the sample ? Or make changes to the sample locally for testing purposes and not check those in? @saghul WDYT? We were hoping to update the sample to use the new method in JitsiLocalTrack.

Reply:
@aaronkvanmeerten Can we get that limitation lifted please?